app/controllers/openid_connect/logout_controller.rb

```diff
@@ -62,6 +63,8 @@ def handle_successful_logout_request(result, redirect_uri)
         }
         @params[:state] = logout_params[:state] if !logout_params[:state].nil?
         @service_provider_name = @logout_form.service_provider&.friendly_name
+        delete_branded_experience(logout: true)
```

Comment:
Should we do this on every branded experience deletion?

Reply:
I'm wondering the same. Looking at all the places where this is used and I don't think I see one where that would be inappropriate to remove the sp session.

Reply:
The one thing it would potentially break is the sp bounce check which gets stored in the SP session (but we've discussed moving it into the user session)

Reply:
@mitchellhenke @jmhooper In the interest of time I'd like to propose keeping this the way it is - leaving `delete_branded_experience` the same for existing use cases that aren't part of logout requests and additionally clearing the `session[:sp]` in logout flows. We can have Katherine refactor after the bug is fixed, I'd just rather not touch the sp bounce code since that's not something I'm familiar with. Does that make sense?